tests/datadir/escape-expression/source/data/in/tables/input-file.csv

```diff
@@ -0,0 +1,3 @@
+"Col1","Col2","Col3"
+"a","b","c"
+"","=foo-bar;-",""
```

Comment:
Ak data obsahuju nejaku hodnotu, ktora zacina s `=`, tak ju OneDrive interpretuje ako vzorec.
A ak je vzorec nespravny, tak to spadne.

My chceme podporovat iba skalarne hodnoty, preto budeme tieto hodnoty escapovat s `'`.